modules/hcp-openstack-autoscale.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-openstack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-openstack-autoscale_{context}"]
+= Enabling node auto-scaling for the hosted cluster
+
+When you need more capacity in your hosted cluster on {rh-openstack-first} and spare agents are available, you can enable auto-scaling to install new worker nodes.
+
+.Procedure
+
+. To enable auto-scaling, enter the following command:
++
+[source,terminal]
+----
+$ oc -n <hosted_cluster_namespace> patch nodepool <hosted_cluster_name> \
+  --type=json \
+  -p '[{"op": "remove", "path": "/spec/replicas"},{"op":"add", "path": "/spec/autoScaling", "value": { "max": 5, "min": 2 }}]'
+----
++
+[NOTE]
+====
+In the example, the minimum number of nodes is 2, and the maximum is 5. The maximum number of nodes that you can add might be bound by your platform. For example, if you use the Agent platform, the maximum number of nodes is bound by the number of available agents.
```

Comment:
Do we want to cross-reference the Agent platform docs?

Reply:
I'll work it in, sure.

Reply:
I'm going to pull the Agent references for now. It was present in the more general templates.